numpy/random/mtrand/mtrand.pyx

```diff
@@ -1372,16 +1372,17 @@ cdef class RandomState:
 
         Return a sample (or samples) from the "standard normal" distribution.
 
-        If positive, int_like or int-convertible arguments are provided,
-        `randn` generates an array of shape ``(d0, d1, ..., dn)``, filled
-        with random floats sampled from a univariate "normal" (Gaussian)
-        distribution of mean 0 and variance 1 (if any of the :math:`d_i` are
-        floats, they are first converted to integers by truncation). A single
-        float randomly sampled from the distribution is returned if no
-        argument is provided.
+        .. note::
+            This is a convenience function for users porting code from Matlab,
+            which wraps `numpy.random.standard_normal`. This inner function
+            takes a tuple to specify the size of the output, which is more
+            consistent with other NumPy functions likes `zeros` and `ones`.
 
-        This is a convenience function.  If you want an interface that takes a
-        tuple as the first argument, use `numpy.random.standard_normal` instead.
+        If positive int_like arguments are provided, `randn` generates an array
+        of shape ``(d0, d1, ..., dn)``, filled
```

Comment:
Might as well justify the text a bit more.

Reply:
Makes the diff noisier and the two commits harder to separate, would prefer not to